src/main/java/com/sap/cloudfoundry/client/facade/adapters/RawInstancesInfo.java

```diff
@@ -28,6 +29,9 @@ public InstancesInfo derive() {
     }
 
     private static List<InstanceInfo> parseProcessStatistics(List<ProcessStatisticsResource> stats) {
+        if (stats == null) {
+            return Collections.emptyList();
+        }
```

Comment:
Is this a valid case?

Reply:
The `states` field of the response object is nullable, so this is just in case